### ISSUE TITLE
[FEATURE] Add Crit roll options

### ISSUE THREAD

**Issue by PaulAshcroft** [Moerill/fvtt-mars-5e#71]
I believe the dnd5e Mess module had an option to maximise crit rolls. other modules that do enable altering crit behaviour (better rolls 5e, etc) aren't compatible (your mod is the best anyway)

**Describe the solution you'd like**
I'd like to have an option to maximise all crit dice (so instead of rolling an extra dice for each damage formula, it just adds the max for the roll). you *could* consider what other mods offer as well (dnd standard, max crit dice, max base dice [no added crit dice], and max all)

**Describe alternatives you've considered**
I tried adding other mods that offer maxing crit dice, but couldnt get any to work (better dice for 5e, Max crit, MidiQol).

**Additional context**
Love the mod! esp. being able to apply the sum of the damage! and toggle crits (great for if you have hexblades curse up!)

**Comment by Moerill** [comment]
Yes Mess had this thanks to a Pull request from someone. I'm currently not fond of adding this feature (or a setting for it).  I may change my mind here, or someone sends a Pull Request implementing this, but no promises. 

(as always i'm open for pull requests for stuff that i'm not fond of implementing, but someone wants.)
